appveyor.yml

```diff
@@ -0,0 +1,20 @@
+environment:
+  matrix:
+  - TARGET: x86_64-pc-windows-msvc
+
+install:
+  - curl -sSf -o rustup-init.exe https://win.rustup.rs/
+  - rustup-init.exe -y --default-host %TARGET%
+  - set PATH=%PATH%;C:\Users\appveyor\.cargo\bin;C:\MinGW\bin
+  - rustc -vV
+  - cargo -vV
+
+build: false
+
+test_script:
+  - cargo test --all
+
+branches:
```

Comment:
I would enable it for all branches.

Reply:
I followed the same way as in travis.yml to avoid running tests twice (for pr and for branch). Removed now.

Reply:
So yeah, we don't really need to run it twice, but we want to run it on every PR and not only after it's merged. So if the filter you have specified here will still trigger a run on a pending PR then it's fine to keep it.